account_move_mail_template_with_subject/data/mail_template_data.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="email_template_send_invoice" model="mail.template">
```

Comment:
You don't need to create new email template for replacing email subject. You can see how to inherit mail template[ ](https://www.cybrosys.com/blog/how-to-inherit-update-an-existing-mail-template-in-odoo-15#:~:text=To%20add%20changes%20to%20the,template%20in%20the%20account%20module.)